doc/GettingStartedKubernetes.md

```diff
@@ -0,0 +1,488 @@
+This page explains how to start a Kubernetes cluster and also run
+Vitess on Kubernetes. This example was most recently tested using
+the binary release of Kubernetes v0.9.1.
+
+## Prerequisites
+
+To complete the exercise in this guide, you must locally install Go 1.3+,
+Vitess' <code>vtctlclient</code> tool, and Google Cloud SDK. The
+following sections explain how to set these up in your environment.
+
+### Install Go 1.3+
+
+You need to install [Go 1.3+](http://golang.org/doc/install) to build the
+<code>vtctlclient</code> tool, which issues commands to Vitess.
+
+After installing Go, make sure your <code>GOPATH</code> environment
+variable is set to the root of your workspace. The most common setting
+is <code>GOPATH=$HOME/go</code>, and the value should identify a
+directory to which your non-root user has write access.
+
+In addition, make sure that <code>$GOPATH/bin</code> is included in
+your <code>$PATH</code>. More information about setting up a Go
+workspace can be found at
+[http://golang.org/doc/code.html#Organization]
+(http://golang.org/doc/code.html#Organization).
+
+## Build and install <code>vtctlclient</code>
+
+The <code>vtctlclient</code> tool issues commands to Vitess.
+
+``` sh
+$ go get git￼￼hub.com/youtube/vitess/go/cmd/vtctlclient
+```
+
+This command downloads and builds the Vitess source code at:
+
+``` sh
+$GOPATH/src/github.com/youtube/vitess/
+```
+
+It also copies the built <code>vtctlclient</code> binary into <code>$GOPATH/bin</code>.
+
+### Set up Google Compute Engine, Container Engine, and Cloud tools
+
+To run Vitess on Kubernetes using Google Compute Engine (GCE),
+you must have a GCE account with billing enabled. The instructions
+below explain how to enable billing and how to associate a billing
+account with a project in the Google Developers Console.
+
+1.  Log in to the Google Developers Console to [enable billing]
+    (https://console.developers.google.com/billing).
+    1.  Click the **Billing** pane if you are not there already.
+    1.  Click **New billing account**
+    1.  Assign a name to the billing account -- e.g. "Vitess on
+        Kubernetes." Then click **Continue**. You can sign up
+        for the free trial to avoid any charges. However, even if you
+        skip the free trial, you should not come anywhere close to
+        exceeding the [App Engine free quota]
```

Comment:
I removed that sentence and instead linked to the page about the Cloud Platform trial. I think we can go back later on and determine whether there's a free quota for GCE. In the meantime, the free trial page confirms that the credit is for any Cloud Platform services, including GCE. Under the free trial, you're limited to 8 concurrent cores.